### DISCUSSION TITLE
Can't I get 'Galaxy Brain' badges?

### DISCUSSION THREAD

**Discussion by kuiux** [community/community#117488] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please let me know if I can't get a 'Galaxy Brazin' badge. I found the following discussions and just wanted to confirm with you all.
https://github.com/orgs/community/discussions/106536

**Comment by Duk3Dum0ntF4n** [comment]
Github staff answered this question in this topic:
https://github.com/orgs/community/discussions/84922

But u still can get this badge, because guy who asked it earned four of them)

**Comment by LiteBrite82** [comment, marked as answer]
Thanks for posting in the GitHub Community, @kuiux!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

With that being said; because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
